source/views/main.js

```diff
@@ -162,7 +162,7 @@ create: function(){
 	this.haveCompass = false;
 	this.compassOpacity = 0;
 	/* Adaptation to Firefox/FirefoxOS */
-	this.transform = (enyo.platform.firefoxOS || enyo.platform.platformName === "firefox") ? "" : "-webkit-";
+	this.transform = ""; //(enyo.platform.firefoxOS || enyo.platform.platformName === "firefox") ? "" : "-webkit-";
```

Comment:
Not sure we want to change it this way, because legacy will need the "-webkit-" for sure. I think we can do with and without. So the condition should be changed to detect LuneOS together with firefox or not to detect LuneOS at all.